src/atoms/Text/Text.stories.js

```diff
@@ -19,4 +19,25 @@ storiesOf('Text', module)
     <Text size={select('Size', sizeOptions, 'large')}>
       {text('Display Text', 'The quick brown fox jumps over the lazy dog')}
     </Text>
+  ))
+  .add('as span', () => (
+    <Text size={select('Size', sizeOptions, 'large')} as="span">
+      {text('Display Text', 'The quick brown fox jumps over the lazy dog')}
+    </Text>
+  ))
+  .add('show content in only 1 line and hide the rest ', () => (
```

Comment:
This should be in the knobs as just a prop you can play with

Reply:
done